.github/workflows/doc-build.yml

```diff
@@ -196,3 +197,36 @@ jobs:
         with:
           name: docs
           path: docs.zip
+
+      - name: Build live doc
+        id: buildlivedoc
+        if: always() && steps.copy.outcome == 'success' && github.ref == 'refs/heads/develop'
```

Comment:
Perhaps should also conditionalize on `github.repository == 'sagemath/sage'` so that it does not run in users' forks if they push a change to their develop (instead of creating a specific branch)

Reply:
OK. Thanks.

Reply:
Do I need `always()` for "Copy live doc" and "Upload live doc" steps?

Reply:
It's better to use `success() || failure()`, see #36617

Reply:
Pending to commit the change `github.repository == 'sagemath/sage' until https://github.com/sagemath/sage/pull/36636#discussion_r1382224796 finish on my own repo.

Reply:
> It's better to use `success() || failure()`, see #36617

OK. 

> Even upon canceling a workflow (manually or automatically when a new commit has been pushed to the same branch), a new step that uses if: always() .... will still be started

But this seems to be a bug on github side.

Reply:
Ready for review.

Reply:
No, this behavior is by design, but one needs to use it sparingly. Essentially only for releasing resources or for saving logs.

Reply:
I see.

Reply:
> Pending to commit the change `github.repository == 'sagemath/sage`

When done, please go ahead and set the label "positive review".

Reply:
Already done. Thanks.